packages/xrpl/src/sugar/autofill.ts

```diff
@@ -260,15 +260,14 @@ export async function calculateFeePerTransactionType(
   tx: Transaction,
   signersCount = 0,
 ): Promise<void> {
-  // netFee is usually 0.00001 XRP (10 drops)
   const netFeeXRP = await getFeeXrp(client)
   const netFeeDrops = xrpToDrops(netFeeXRP)
   let baseFee = new BigNumber(netFeeDrops)
 
   // EscrowFinish Transaction with Fulfillment
   if (tx.TransactionType === 'EscrowFinish' && tx.Fulfillment != null) {
     const fulfillmentBytesSize: number = Math.ceil(tx.Fulfillment.length / 2)
```

Comment:
Do you know why we are dividing the `tx.Fulfillment.length` by `2` ? I know that you didn't write this code, but I didn't understand it.

Reply:
This is from the docs:

> The minimum [transaction cost](https://xrpl.org/docs/concepts/transactions/transaction-cost/) to submit an EscrowFinish transaction increases if it contains a fulfillment. If the transaction has no fulfillment, the transaction cost is the standard 10 drops. If the transaction contains a fulfillment, the transaction cost is 330 [drops of XRP](https://xrpl.org/docs/references/protocol/data-types/basic-data-types/#specifying-currency-amounts) plus another 10 drops for every 16 bytes in size of the preimage.

The division by 2 is due to the fact that each byte is represented by 2 characters in hex

Reply:
ok thanks